.github/sync.yml

```diff
@@ -9,6 +9,7 @@ group:
       WordPress/openverse-api
       WordPress/openverse-frontend
       WordPress/openverse-browser-extension
+      WordPress/openverse-infrastructure
```

Comment:
This includes the infrastructure repo in the release drafter, I think we're not ready to include it yet (?) Not sure. CC @zackkrida

Reply:
Agreed! In fact, I think that'd be desirable :smile: